chain/consensus/mir/state_manager.go

```diff
@@ -118,7 +119,9 @@ func (sm *StateManager) RestoreState(checkpoint *checkpoint.StableCheckpoint) er
 
 	for e, membership := range config.Memberships {
 		// skew membership to current epoch, we are starting from a checkpoint
-		sm.memberships[t.EpochNr(e)+sm.currentEpoch] = make(map[t.NodeID]t.NodeAddress)
+		// sm.memberships[t.EpochNr(e)+sm.currentEpoch] = make(map[t.NodeID]t.NodeAddress)
```

Comment:
remove? Also, does this work when restarting a validator? I had to add this workaround to populate the membership info when we started from a checkpoint instead of from scratch.

Reply:
It is not removed. It is actually the same. I just added the line with `make` for the map. But I could return it back for readability.
Fixed.